packages/react/src/components/CTA/ButtonCTA.js

```diff
@@ -67,7 +67,9 @@ const _renderButtons = ({
       let title = videoTitle.filter(name => {
         return name.key === key;
       });
-      button.copy = !title[0] ? button.copy : title[0].title;
+      button.copy = !title[0]
+        ? button.copy
+        : `${title[0].title} ${title[0].duration}`;
```

Comment:
g11n; I'd add another prop for this kind of composite string, given it's hard to assume order of contents when different languages are in consideration:


```suggestion
        : formatButtonCopyWithTitle({ title: title[0].title, duration: title[0].duration });
```

```javascript
ButtonCTA.defaultProps = {
  formatButtonCopyWithTitle: ({ title, duration }) => `${title[0].title} ${title[0].duration}`;
};
```

Another approach may be letting `useVideoData()` set `title` with duration (meaning, relegating formatting logic to the caller (parent) of `<ButtonCTA>`, etc.), though I'm not sure if it's safe to do so (do we have some code that requires distinction between title and duration)?

Reply:
Originally I had it set that this logic of concatenating the `title` and `duration` in `useVideoData()` but for CardCTA specifically we need to separate the two based on the designs.

You brought up a good point, I'll make changes to taken in another prop